[2024-02-01T12:56:24.089+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: message_display_dag.display_second_message manual__2024-02-01T12:55:48.893644+00:00 [queued]>
[2024-02-01T12:56:24.194+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: message_display_dag.display_second_message manual__2024-02-01T12:55:48.893644+00:00 [queued]>
[2024-02-01T12:56:24.195+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-02-01T12:56:24.263+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): display_second_message> on 2024-02-01 12:55:48.893644+00:00
[2024-02-01T12:56:24.307+0000] {standard_task_runner.py:60} INFO - Started process 16631 to run task
[2024-02-01T12:56:24.361+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'message_display_dag', 'display_second_message', 'manual__2024-02-01T12:55:48.893644+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/quick_dag.py', '--cfg-path', '/tmp/tmphh3m3cjw']
[2024-02-01T12:56:24.391+0000] {standard_task_runner.py:88} INFO - Job 17: Subtask display_second_message
[2024-02-01T12:56:24.872+0000] {task_command.py:423} INFO - Running <TaskInstance: message_display_dag.display_second_message manual__2024-02-01T12:55:48.893644+00:00 [running]> on host 66086c79257b
[2024-02-01T12:56:25.191+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='message_display_dag' AIRFLOW_CTX_TASK_ID='display_second_message' AIRFLOW_CTX_EXECUTION_DATE='2024-02-01T12:55:48.893644+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-01T12:55:48.893644+00:00'
[2024-02-01T12:56:25.200+0000] {logging_mixin.py:188} INFO - This is the 2nd message (2/3)ðŸŸ¡
[2024-02-01T12:56:25.201+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-01T12:56:25.242+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=message_display_dag, task_id=display_second_message, execution_date=20240201T125548, start_date=20240201T125624, end_date=20240201T125625
[2024-02-01T12:56:25.349+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-01T12:56:25.418+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
